internal/backends/winit/renderer/femtovg.rs

```diff
@@ -38,7 +38,9 @@ impl super::WinitCompatibleRenderer for GlutinFemtoVGRenderer {
             #[cfg(not(target_arch = "wasm32"))]
             opengl_context,
             #[cfg(target_arch = "wasm32")]
-            winit_window.canvas(),
+            winit_window
+                .canvas()
+                .ok_or_else(|| "FemtoVG Renderer: winit didn't return a canvas")?,
```

Comment:
I don't know if this error can happen in practice.

Reply:
I think this could happen if the wasm module is used in a worker thread. Doubt it happens in practice for us.